plugin/storage/grpc/proto/storage.proto

```diff
@@ -148,6 +148,63 @@ message FindTraceIDsResponse {
     ];
 }
 
+message Throughput {
```

Comment:
Schema should be documented. Wherever you are copying this from, we have comments explaining all types.